docs/other-topics/upgrade.md

```diff
@@ -315,6 +315,15 @@ User.belongsToMany(Country, {
 - A few bugs in how indexes were named have been fixed. This means your index names could change.
 
 ## Minor Breaking changes
+### Renamed APIs
+- `Sequelize.QueryInterface` has been renamed to `Sequelize.AbstractQueryInterface`. Previously, you could do: 
```

Comment:
I wouldn't add the `Sequelize.` prefix. We don't want people to access these values on the constructor, that's just there for legacy reasons

Since another PR has been merged, let's add `ModelColumnAttributeOptions` to the list

```suggestion
The following exports have been renamed:

- `QueryInterface` has been renamed to `AbstractQueryInterface`.
- `ModelColumnAttributeOptions` has been renamed to `AttributeOptions`.
```